internal/local/configure.go

```diff
@@ -18,13 +18,26 @@ import (
 
 func (service *ProvisioningService) Configure() utils.ReturnCode {
 	service.setupWsmanClient("admin", service.flags.Password)
-
-	if service.flags.SubCommand == utils.SubCommandAddWifiSettings {
+	switch service.flags.SubCommand {
+	case utils.SubCommandAddWifiSettings:
 		return service.AddWifiSettings()
+	case utils.SubCommandEnableWifiPort:
+		return service.EnableWifiPort()
+	default:
 	}
 	return utils.IncorrectCommandLineParameters
 }
 
+func (service *ProvisioningService) EnableWifiPort() utils.ReturnCode {
+	rc := service.EnableWifi()
+	if rc != utils.Success {
+		log.Error("Failed enabling wifi port and local profile synchronization.  Return Code: ", rc)
```

Comment:
Since the code is returned to the OS, we haven't made of putting it in the log messages. I think "Failed to enable ..." reads a bit better than "Failed enabling..."

Reply:
Done